site/docs/guides/contributor/markdown.md

```diff
@@ -24,7 +24,7 @@ Scroll to the bottom of the page to see the contents of the README.md displayed.
 These pages are also used to create Egeria's webpages using a technology called **GitHub pages**.
 Click on the next link to display the tools directory in the website.
 
-* [https://odpi.github.io/guides/contributor/markdown](https://github.com/odpi/blob/main/site/docs/guides/contributor/markdown)
+* [https://egeria-project.org/guides/contributor/markdown](https://github.com/odpi/blob/main/site/docs/guides/contributor/markdown)
```

Comment:
the link does not match the text and the link is an invalid url

Reply:
Keeping with the internal link idea- this should also be an internal link

Reply:
Agreed on it not matching and also on the internal link - a little unclear on the invalid URL as when I click on it it takes me to the right page: [https://egeria-project.org/guides/contributor/markdown]( https://egeria-project.org/guides/contributor/markdown)

Reply:
Ah wait actually on a second glance at this page it references GitHub pages and is perhaps slightly too meta to be fixable by 1 link - this page needs editing with the correct context and also the rest of the links on this page are not great either. Will get this whole page working a little cleaner. On that I do think there is an argument to be made for the links on here to not be internal links given this page is, by design, intended to also be viewed in the context of the github page: `"click on this link to display the tools directory where this file lives on GitHub"`

Reply:
Will move this section to a fresh issue as it's out of scope for this one